prow/plugins/label/add_label.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package label
+
+import (
+	"github.com/Sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"regexp"
+	"strings"
+)
+
+const pluginName = "add_label"
+
+var (
+	COMMAND_NAME    = "label"
```

Comment:
It's unusual to see `CAPITALIZED_CONSTANTS` in go. Since this is only used once, I would just remove it and make the next line easier to read.